CavalcanteLucas/labdale
# Feature/OL45-todo_is_done

**Link to task:**
https://labcodes.atlassian.net/browse/OL-211

**How to test:**
With a user logged in, create a TodoList, then create a Todo. After that, mark a the just created Todo as "complete" marking a checkbox besides its title. See if a dismissable notification of success appear on the DashboardPage when operation is complete.

**Description of your solution:**
This PR inlcudes a 'is_done' boolean field on the Todo model, and implements the backend API and the frontend interfaces that enable Todos to be individually marked as "complete". When marking a Todo from done to undone or vice-versa, a Notification of success should appear on the topscreen of the dashboard. Tests that covert PUT requests to Todos are updated accordingly.

**Observations**
This PR should be merged on master branch after merge of [PR17](https://github.com/CavalcanteLucas/labdale/pull/17)
